.circleci/initialize-manager.py

```diff
@@ -18,10 +18,11 @@ def initialize_manager(max_runtime):
 
     # Catch any exception that occurs so that we can gracefully teardown
     try:
-        put(ci_workdir + "/scripts/machine-launch-script.sh", manager_home_dir)
+	# wait until machine launch is complete
+        with cd(manager_home_dir):
+            run("timeout 10m grep -q '.*machine launch script complete.*' <(tail -f machine-launchstatus)")
```

Comment:
Unlike the previous version, this won't produce output that is debuggable by looking at circleci web ui, correct? if that's the case, can you perhaps `tail` or `cat` the log before exiting on failure?